configure.ac

```diff
@@ -1389,4 +1389,19 @@ m4_ifdef([project_orte], [ORTE_CONFIG_FILES])
 m4_ifdef([project_ompi], [OMPI_CONFIG_FILES])
 m4_ifdef([project_oshmem], [OSHMEM_CONFIG_FILES])
 
+OPAL_UNIQ([opal_libnl_libs])
+OPAL_UNIQ([opal_libnl3_libs])
+AC_MSG_CHECKING([for libraries that use libnl])
+AS_IF([test -n "$opal_libnl_libs"],
+      [AC_MSG_RESULT([$opal_libnl_libs])],
+      [AC_MSG_RESULT([(none)])])
+AC_MSG_CHECKING([for libraries that use libnl3])
+AS_IF([test -n "$opal_libnl3_libs"],
+      [AC_MSG_RESULT([$opal_libnl3_libs])],
+      [AC_MSG_RESULT([(none)])])
+AS_IF([test -n "$opal_libnl_libs" && test -n "$opal_libnl3_libs"],
+      [AC_MSG_WARN([Both libnl and libnl3 are required])
+       AC_MSG_WARN([This is a configuration that is known to cause run-time crashes])])
```

Comment:
I would add an AC_ERROR so that configure aborts.  We know that this will crash -- best to make configure abort and let the user figure it out (vs. a configure warning that scrolls by / is ignored, and then the user is confused later when all MPI applications crash).